keen/tests/client_tests.py

```diff
@@ -214,8 +206,14 @@ def url_escape(self, url):
             return urllib.parse.quote(url)
 
 
-
+@patch("requests.Session.get")
 class QueryTests(BaseTestCase):
+
+    int_response = MockedRequest(status_code=200, json_response=2)
```

Comment:
Having these up here is a really cool idea! I might put them in all-caps (e.g., `INT_RESPONSE`) to indicate that they’re invariant.